postcss.config.js

```diff
@@ -1,7 +1,20 @@
+const autoprefixer = require('autoprefixer');
+const assets = require('postcss-assets');
+
+const CF_DIST = process.env.CLOUDFRONT_DIST;
```

Comment:
That's a good thought that I hadn't considered, but even if you're correct, this pull request is still a step in the right direction. Maybe we can file a follow-up issue for this, so that it doesn't block this PR. (I'd still be curious to hear from @blarghmatey or @shaidar, though.)